src/ai/susi/server/api/cms/ListSkillService.java

```diff
@@ -43,7 +43,7 @@ public ServiceResponse serviceImpl(Query call, HttpServletResponse response, Aut
 
         String model_name = call.get("model", "general");
         File model = new File(DAO.model_watch_dir, model_name);
-        String group_name = call.get("group", "Knowledge");
+        String group_name = call.get("group", "All");
```

Comment:
Does this return all items when the `Query.group` value associated in the call is empty? If so, this could potentially become problematic as the number of skills increase. It'd always be good to send all but in a paginated fashion, maybe only display the top 30-40 items and when the client explicitly requests a reload, then load the next 30-40 etc..,

So far, I don't see a problem with this but it might need to be fixed.

Reply:
@sudheesh001 I see your point. I guess we can implement that in future PRs :-)

Reply:
Sure.